packages/app/src/settings/actions/toggle-update-available.action.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import { SettingsEvent } from "App/settings/constants"
+import { checkAppUpdateFlowToShow } from "App/modals-manager/actions"
+
+export const toggleUpdateAvailable = createAsyncThunk<boolean, boolean>(
+  SettingsEvent.ToggleUpdateAvailable,
+  async (payload, { dispatch }) => {
+    dispatch(checkAppUpdateFlowToShow())
```

Comment:
Not 100% sure, but the original code contained such a line `        dispatch.settings.update({ appUpdateAvailable })` 

Is it ok that this is skipped?